src/main/java/mate/academy/bookstore/model/Book.java

```diff
@@ -44,4 +51,13 @@ public class Book {
 
     @Column(nullable = false)
     private boolean isDeleted = false;
+
+    @ManyToMany(fetch = FetchType.LAZY)
+    @JoinTable(
+            name = "book_category",
+            joinColumns = @JoinColumn(name = "book_id"),
+            inverseJoinColumns = @JoinColumn(name = "category_id"))
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    private Set<Category> categories = new HashSet<>();
```

Comment:
@ToString(exclude =
@EqualsAndHashCode(exclude = 
on class level, so you don't repeat those annotations for each and every field